cfme/tests/intelligence/reports/test_reports.py

```diff
@@ -534,6 +534,33 @@ def test_reports_filter_content(
     assert sorted(expected) == sorted(got)
 
 
+@test_requirements.filtering
+@pytest.mark.meta(automates=[1696412])
+def test_reports_filter_expression_editor_disk_size(appliance, request, soft_assert, get_report):
```

Comment:
we are not using this `soft_assert` in test.

Reply:
yes, forgot to delete it, thank you :)